From d5dde7ca91a5aed273d8fe269e1a5194e85c8c79 Mon Sep 17 00:00:00 2001
From: Scott Garman <scott.a.garman@intel.com>
Date: Tue, 13 Jul 2010 16:46:46 +0800
Subject: [PATCH] apmd: upgrade to 3.2.2-14

Add by RP to address "unable to infer tagged configuration" error:
   commit 35de05e61b88c0808a5e885bb0efdf420555d5ad
   Author: Richard Purdie <rpurdie@rpsys.net>
   Date:   Sun Jun 1 16:13:38 2008 +0000

   apmd: Use libtool --tag options to avoid problems with libtool 2.2.4 (from poky)

However I didn't see same issue with current libtool-2.2.10. Also per my understanding,
the default tag, if not specified, falls back to CC. So disable it from patching, but
keep it here. If we encounter similar issue in the future, we could then push upstream

Comment added by Kevin Tian <kevin.tian@intel.com>, 2010-07-16

Upstream-Status: Pending

Signed-off-by: Scott Garman <scott.a.garman@intel.com>

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 92fc0fd..8e283dc 100644
--- a/Makefile
+++ b/Makefile
@@ -59,8 +59,8 @@ RANLIB=ranlib
 #LDFLAGS=-s
 
 LIBTOOL=libtool --quiet
-LT_COMPILE = $(LIBTOOL) --mode=compile $(CC)
-LT_LINK = $(LIBTOOL) --mode=link $(CC)
+LT_COMPILE = $(LIBTOOL) --tag=CC --mode=compile $(CC)
+LT_LINK = $(LIBTOOL)  --tag=CC --mode=link $(CC)
 LT_INSTALL = $(LIBTOOL) --mode=install install
 LT_CLEAN = $(LIBTOOL) --mode=clean rm
 
